tests/tests.js

```diff
@@ -24,7 +24,7 @@ function runTests(data, excused, flags, features) {
       });
     }
 
-    if (generated !== expected && !isError) {
+    if (true) {//generated !== expected && !isError) {
```

Comment:
Is there a reason not to remove the `if` altogether?

Reply:
Not from my point of view.

Reply:
Then let's remove it.

Reply:
I already did :)